README.md

```diff
@@ -1,15 +1,17 @@
-# SIG: Web - website2017
-This is [CompSoc Edinburgh](https://comp-soc.com)'s website, built by SIG: Web.
+# SIG: Web - website2020 (up to date!) - a super duper amazing project
+This is [CompSoc Edinburgh](https://comp-soc.com)'s website, built by SIG:Web.
 
-## How to test
-It uses Jekyll. It also works nicely with Docker.
+## Running on your computer
+It uses Jekyll. It also works very amazingly nicely with Docker.
 
-If you use Linux or macOS, run `docker-run.sh`. It will automatically build, server, and watch the directory for changes. It will listen on port 4000.
+If you use Linux or macOS, run `docker-run.sh`. It will automatically build, server, and watch the directory for changes and it will listen on port 4000. 
 
-If you're on Windows, please contribute some instructions. You'll hopefully be able to read `docker-run.sh` and adapt the commands to play nice with docker-machine, or native Docker if that's your thing.
+If you're on Windows (I'm sorry), please contribute some instructions. You'll hopefully be able to read `docker-run.sh` and adapt the commands to play nice with docker-machine, or native Docker if that's your thing.
 
-### Running without Docker
+## Running on your computer without Docker
 If you have `bundler` installed, type:
 
 - `bundle`, to install the Gemfile packages
 - `bundle exec jekyll serve` to host a local server that will watch the directory for changes
+
+Thanks for visiting!!
```

Comment:
Sorry, but this double exclamation mark clearly breaches CompSoc's strategic exclamation mark reserve. Please reduce this to a single bang or you shall be held accountable.

Reply:
@neanias I'll reduce you to a single bang

Reply:
@findoslice I thought you'd never asl